File: app/src/main/java/com/permissionnanny/ConfirmRequestActivity.java
package com.permissionnanny;

import android.os.Bundle;
import javax.inject.Inject;

/**
 *
 */
public class ConfirmRequestActivity extends BaseActivity {

    @Inject ConfirmRequestBinder mBinder;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        getComponent().inject(this);
        mBinder.preOnCreate(savedInstanceState);
        super.onCreate(savedInstanceState);
        mBinder.onCreate(savedInstanceState);
    }

    @Override
    public void onBackPressed() {
        super.onBackPressed();
        mBinder.onBackPressed();
    }
}

File: app/src/main/kotlin/com/permissionnanny/ConfirmRequestActivity.kt
package com.permissionnanny

import android.os.Bundle
import javax.inject.Inject

/**

 */
class ConfirmRequestActivity : BaseActivity() {

    @Inject internal lateinit var binder: ConfirmRequestBinder

    override fun onCreate(savedInstanceState: Bundle?) {
        getComponent().inject(this)
        binder.preOnCreate(savedInstanceState)
        super.onCreate(savedInstanceState)
        binder.onCreate(savedInstanceState)
    }

    override fun onBackPressed() {
        super.onBackPressed()
        binder.onBackPressed()
    }
}
